src/Socket/messages-recv.ts

```diff
@@ -909,7 +909,7 @@ export const makeMessagesRecvSocket = (config: SocketConfig) => {
 		}
 
 		// delete data once call has ended
-		if(status === 'reject' || status === 'accept' || status === 'timeout') {
+		if(status === 'reject' || status === 'timeout' || status === 'terminate' || status === 'accept') {
```

Comment:
```suggestion
		if(status === 'reject' || status === 'accept' || status === 'timeout' || status === 'terminate' ) {
```
just a cleaner changelog